### ISSUE TITLE
How to bypass proxy on a per context basis when one is defined for the browser globally

### ISSUE THREAD

**Issue by Paikan** [microsoft/playwright-python#603]
I am working on a project exposing playwright functionalities via a REST service.

Basically the client can provide a url and eventually a proxy server as input and gets the html content as response.

If I understand well according to the documentation I can define proxy settings on a per context basis with the following code:

```python
browser = await chromium.launch(proxy={"server": "per-context"})
context = await browser.new_context(proxy={"server": "http://myproxy.com:3128"})
```

The problem I have is that I can't find a way to disable proxy usage on a new context (I have tried to use an empty server value).

Basically once I have defined proxy on launch (which seems mandatory for my use case) I don't know how a client can bypass proxy completely on a per request basis.

Have I missed something?

Thanks

**Comment by Paikan** [comment]
I guess I can use the bypass setting with the domain of the requested url in that case. 

Closing the issue.
